{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/public/views.py

```diff
@@ -75,3 +83,97 @@ def about():
     """About page."""
     form = LoginForm(request.form)
     return render_template("public/about.html", form=form)
+{% if cookiecutter.use_google_logging == "yes" %}
+
+def get_google_provider_cfg():
+    """Returns Google Provide configuration."""
+    return requests.get(current_app.config["GOOGLE_DISCOVERY_URL"]).json()
```

Comment:
Maybe would be a good idea to check the request code and raise appropriate exceptions if a 2xx isn't received before returning the JSON.